A/AbstractXsdTypes/Versions.toml

```diff
@@ -1,2 +1,2 @@
 ["0.1.0"]
-git-tree-sha1 = "9b0214aa1bdb299d8a65dc6f25678322f35b2795"
+git-tree-sha1 = "7a644fc2d9a859a953b5fa106f663312cdeb6deb"
```

Comment:
I'm not sure you need to change the commit of the registered version.

Reply:
probably a good idea since the uuid is commited in the project.toml

Reply:
That was indeed my reasoning for updating the commit.

Reply:
Uhm, but this should be the git-tree-hash, not the commit has itself.

Reply:
My bad, copied the wrong value.